File: Treasury.Domain/Model/OrganizationComment.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace Treasury.Domain.Model
{
    public partial class OrganizationComment
    {
        public int Id { get; set; }
        public string NameOfClub { get; set; }
        public DateTime CommentDate { get; set; }
        public string Comment { get; set; }
        public DateTime Timestamp { get; set; }

        public virtual Organization NameOfClubNavigation { get; set; }
    }
}

File: src/main/java/edu/wpi/sga/treasury/domain/model/OrganizationComment.java
package edu.wpi.sga.treasury.domain.model;

import lombok.Getter;
import lombok.Setter;

import javax.persistence.*;
import java.time.LocalDate;
import java.time.LocalDateTime;

@Getter
@Setter
@Entity
@Table(name = "organization_comment")
public class OrganizationComment {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", nullable = false)
    private Integer id;

    @ManyToOne(fetch = FetchType.LAZY, optional = false)
    @JoinColumn(name = "organization_id", nullable = false)
    private Organization organization;

    @Column(name = "comment_date", nullable = false)
    private LocalDate commentDate;

    @Column(name = "comment", nullable = false)
    private String comment;

    @Column(name = "last_modified", nullable = false)
    private LocalDateTime lastModified;

}